react-swipe.js

```diff
@@ -72,8 +72,16 @@
     render: function() {
       return React.createElement('div', React.__spread({}, this.props, {style: styles.container}),
         React.createElement('div', {style: styles.wrapper},
-          React.Children.map(this.props.children, function (child) {
-            return React.addons.cloneWithProps(child, {style: styles.child});
+          React.Children.map(this.props.children, function (child,index) {
+
+            function merge_options(obj1,obj2) {
```

Comment:
@andykenward I would prefer to use [object-assign](https://github.com/sindresorhus/object-assign) instead.